include/libtorrent/alert_types.hpp

```diff
@@ -2934,6 +2934,27 @@ TORRENT_VERSION_NAMESPACE_3
 		aux::noexcept_movable<tcp::endpoint> ip;
 	};
 
+	// posted when a prioritize_files() or file_priority() update of the file
+	// priorities complete, which requires a round-trip to the disk thread.
+	//
+	// If the disk operation fails, ``operation_t`` and ``error`` indicates what
```

Comment:
Seems it isn't implemented...

Reply:
oops. my first pass had error states, but there's already a `file_error_alert` being posted in this case